### ISSUE TITLE
Not visible, shifted and overlapped controls

### ISSUE THREAD

**Issue by falinsky** [unclebob/spacewar#8]
When the screen is less than 2000x1150 a few controls and indicators are not visible, shifted or overlapped by other ones.
![space war](https://user-images.githubusercontent.com/859638/49857102-52f15880-fdfa-11e8-9028-72183937e1f2.png)

**Comment by unclebob** [comment]
Yes.  While the positions of most of the elements are calculated to be proportional to the screen size; a few are absolute.  If you are playing the ClojureScript version of the game then you can simply adjust the magnification of your browser window.  On the desktop you need a resolution that fits.

Would you care to fix this?  Send me a pull request.
